signature/algorithm_test.go

```diff
@@ -229,3 +229,19 @@ func TestSignatureAlgorithm(t *testing.T) {
 		})
 	}
 }
+
+func TestKeyTypeStringer(t *testing.T) {
+	if KeyTypeEC.String() != "ECDSA" {
+		t.Fatal("KeyTypeEC stringer test failed")
+	}
+
+	if KeyTypeRSA.String() != "RSA" {
+		t.Fatal("KeyTypeRSA stringer test failed")
+	}
+
+	var keyType KeyType
+	keyType = 10
```

Comment:
I suggest use a table driven test for this test case

Reply:
Done.